{"uid":"f4d7de015166ef2d","name":"test_fail_and_rollback","fullName":"tests.integration.ha_tests.test_rollback_to_master_label#test_fail_and_rollback","historyId":"f474748990346242c7ea0dc739477429","time":{"start":1733797150257,"stop":1733797664852,"duration":514595},"status":"failed","statusMessage":"AssertionError: assert 0 == 1\n +  where 0 = <built-in function countOf>(dict_values(['replica', 'replica', 'primary']), 'master')\n +    where <built-in function countOf> = operator.countOf\n +    and   dict_values(['replica', 'replica', 'primary']) = <built-in method values of dict object at 0x7f8f85242200>()\n +      where <built-in method values of dict object at 0x7f8f85242200> = {'postgresql-k8s/0': 'replica', 'postgresql-k8s/1': 'replica', 'postgresql-k8s/2': 'primary'}.values","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 880, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/tests/integration/ha_tests/test_rollback_to_master_label.py\", line 167, in test_fail_and_rollback\n    assert operator.countOf(instances_roles.values(), \"master\") == 1\nAssertionError: assert 0 == 1\n +  where 0 = <built-in function countOf>(dict_values(['replica', 'replica', 'primary']), 'master')\n +    where <built-in function countOf> = operator.countOf\n +    and   dict_values(['replica', 'replica', 'primary']) = <built-in method values of dict object at 0x7f8f85242200>()\n +      where <built-in method values of dict object at 0x7f8f85242200> = {'postgresql-k8s/0': 'replica', 'postgresql-k8s/1': 'replica', 'postgresql-k8s/2': 'primary'}.values","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1733796599034,"stop":1733796599035,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1733796599035,"stop":1733796599292,"duration":257},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1733796599035,"stop":1733796599035,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1733797150256,"stop":1733797150256,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"continuous_writes","time":{"start":1733797150256,"stop":1733797150257,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"failed","statusMessage":"AssertionError: assert 0 == 1\n +  where 0 = <built-in function countOf>(dict_values(['replica', 'replica', 'primary']), 'master')\n +    where <built-in function countOf> = operator.countOf\n +    and   dict_values(['replica', 'replica', 'primary']) = <built-in method values of dict object at 0x7f8f85242200>()\n +      where <built-in method values of dict object at 0x7f8f85242200> = {'postgresql-k8s/0': 'replica', 'postgresql-k8s/1': 'replica', 'postgresql-k8s/2': 'primary'}.values","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 880, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/tests/integration/ha_tests/test_rollback_to_master_label.py\", line 167, in test_fail_and_rollback\n    assert operator.countOf(instances_roles.values(), \"master\") == 1\nAssertionError: assert 0 == 1\n +  where 0 = <built-in function countOf>(dict_values(['replica', 'replica', 'primary']), 'master')\n +    where <built-in function countOf> = operator.countOf\n +    and   dict_values(['replica', 'replica', 'primary']) = <built-in method values of dict object at 0x7f8f85242200>()\n +      where <built-in method values of dict object at 0x7f8f85242200> = {'postgresql-k8s/0': 'replica', 'postgresql-k8s/1': 'replica', 'postgresql-k8s/2': 'primary'}.values","steps":[],"attachments":[{"uid":"ad4197b93366da75","name":"log","source":"ad4197b93366da75.txt","type":"text/plain","size":10611}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"continuous_writes::finalizer","time":{"start":1733797664854,"stop":1733797670904,"duration":6050},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail::0","time":{"start":1733797670904,"stop":1733797670905,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1733797670905,"stop":1733797671304,"duration":399},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1733797671304,"stop":1733797671305,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1733797671305,"stop":1733797671305,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1733797671305,"stop":1733797671305,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"@pytest.mark.skipif(False, reason='Requires amd64 architecture')"},{"name":"tag","value":"unstable"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_rollback_to_master_label"},{"name":"host","value":"github-runner"},{"name":"thread","value":"19411-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_rollback_to_master_label"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"5d7dc588b6e7c209","status":"unknown","time":{}}],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":43,"broken":0,"skipped":27,"passed":0,"unknown":5,"total":75},"items":[{"uid":"d9b7cf6739b949fe","reportUrl":"../263//#testresult/d9b7cf6739b949fe","status":"failed","statusDetails":"AssertionError: assert 0 == 1\n +  where 0 = <built-in function countOf>(dict_values(['replica', 'replica', 'primary']), 'master')\n +    where <built-in function countOf> = operator.countOf\n +    and   dict_values(['replica', 'replica', 'primary']) = <built-in method values of dict object at 0x7a33635d3000>()\n +      where <built-in method values of dict object at 0x7a33635d3000> = {'postgresql-k8s/0': 'replica', 'postgresql-k8s/1': 'replica', 'postgresql-k8s/2': 'primary'}.values","time":{"start":1733711176159,"stop":1733711690878,"duration":514719}},{"uid":"f47d2c2d54768191","reportUrl":"../262//#testresult/f47d2c2d54768191","status":"failed","statusDetails":"AssertionError: assert 0 == 1\n +  where 0 = <built-in function countOf>(dict_values(['replica', 'replica', 'primary']), 'master')\n +    where <built-in function countOf> = operator.countOf\n +    and   dict_values(['replica', 'replica', 'primary']) = <built-in method values of dict object at 0x7909ad95f6c0>()\n +      where <built-in method values of dict object at 0x7909ad95f6c0> = {'postgresql-k8s/0': 'replica', 'postgresql-k8s/1': 'replica', 'postgresql-k8s/2': 'primary'}.values","time":{"start":1733622927133,"stop":1733623552913,"duration":625780}},{"uid":"5331732c84fef429","reportUrl":"../261//#testresult/5331732c84fef429","status":"failed","statusDetails":"AssertionError: assert 0 == 1\n +  where 0 = <built-in function countOf>(dict_values(['replica', 'replica', 'primary']), 'master')\n +    where <built-in function countOf> = operator.countOf\n +    and   dict_values(['replica', 'replica', 'primary']) = <built-in method values of dict object at 0x7cc6a7f810c0>()\n +      where <built-in method values of dict object at 0x7cc6a7f810c0> = {'postgresql-k8s/0': 'replica', 'postgresql-k8s/1': 'replica', 'postgresql-k8s/2': 'primary'}.values","time":{"start":1733537641906,"stop":1733538198540,"duration":556634}},{"uid":"3a182adc690417fb","reportUrl":"../260//#testresult/3a182adc690417fb","status":"failed","statusDetails":"AssertionError: assert 0 == 1\n +  where 0 = <built-in function countOf>(dict_values(['replica', 'replica', 'primary']), 'master')\n +    where <built-in function countOf> = operator.countOf\n +    and   dict_values(['replica', 'replica', 'primary']) = <built-in method values of dict object at 0x78f5f3755980>()\n +      where <built-in method values of dict object at 0x78f5f3755980> = {'postgresql-k8s/0': 'replica', 'postgresql-k8s/1': 'replica', 'postgresql-k8s/2': 'primary'}.values","time":{"start":1733452160822,"stop":1733452715997,"duration":555175}},{"uid":"8f8a8808b5810fe3","reportUrl":"../259//#testresult/8f8a8808b5810fe3","status":"failed","statusDetails":"AssertionError: assert 0 == 1\n +  where 0 = <built-in function countOf>(dict_values(['replica', 'replica', 'primary']), 'master')\n +    where <built-in function countOf> = operator.countOf\n +    and   dict_values(['replica', 'replica', 'primary']) = <built-in method values of dict object at 0x737cce97b900>()\n +      where <built-in method values of dict object at 0x737cce97b900> = {'postgresql-k8s/0': 'replica', 'postgresql-k8s/1': 'replica', 'postgresql-k8s/2': 'primary'}.values","time":{"start":1733369720474,"stop":1733370303608,"duration":583134}},{"uid":"c75c3d7c02b859e6","reportUrl":"../258//#testresult/c75c3d7c02b859e6","status":"failed","statusDetails":"AssertionError: assert 0 == 1\n +  where 0 = <built-in function countOf>(dict_values(['replica', 'replica', 'primary']), 'master')\n +    where <built-in function countOf> = operator.countOf\n +    and   dict_values(['replica', 'replica', 'primary']) = <built-in method values of dict object at 0x75852d7c18c0>()\n +      where <built-in method values of dict object at 0x75852d7c18c0> = {'postgresql-k8s/0': 'replica', 'postgresql-k8s/1': 'replica', 'postgresql-k8s/2': 'primary'}.values","time":{"start":1733277241231,"stop":1733277754232,"duration":513001}},{"uid":"9fdb836d8a1da705","reportUrl":"../257//#testresult/9fdb836d8a1da705","status":"failed","statusDetails":"AssertionError: assert 0 == 1\n +  where 0 = <built-in function countOf>(dict_values(['replica', 'replica', 'primary']), 'master')\n +    where <built-in function countOf> = operator.countOf\n +    and   dict_values(['replica', 'replica', 'primary']) = <built-in method values of dict object at 0x791263197400>()\n +      where <built-in method values of dict object at 0x791263197400> = {'postgresql-k8s/0': 'replica', 'postgresql-k8s/1': 'replica', 'postgresql-k8s/2': 'primary'}.values","time":{"start":1733194905707,"stop":1733195518061,"duration":612354}},{"uid":"b93805c67f18601b","reportUrl":"../256//#testresult/b93805c67f18601b","status":"failed","statusDetails":"AssertionError: assert 0 == 1\n +  where 0 = <built-in function countOf>(dict_values(['replica', 'replica', 'primary']), 'master')\n +    where <built-in function countOf> = operator.countOf\n +    and   dict_values(['replica', 'replica', 'primary']) = <built-in method values of dict object at 0x758c945fc340>()\n +      where <built-in method values of dict object at 0x758c945fc340> = {'postgresql-k8s/0': 'replica', 'postgresql-k8s/1': 'replica', 'postgresql-k8s/2': 'primary'}.values","time":{"start":1733106744145,"stop":1733107420936,"duration":676791}},{"uid":"be06421e544860f","reportUrl":"../255//#testresult/be06421e544860f","status":"unknown","time":{}},{"uid":"c7dbb627fad945a","reportUrl":"../254//#testresult/c7dbb627fad945a","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1732936150326,"stop":1732936150326,"duration":0}},{"uid":"cb4cefc52295e7a2","reportUrl":"../253//#testresult/cb4cefc52295e7a2","status":"failed","statusDetails":"AssertionError: assert 0 == 1\n +  where 0 = <built-in function countOf>(dict_values(['replica', 'replica', 'primary']), 'master')\n +    where <built-in function countOf> = operator.countOf\n +    and   dict_values(['replica', 'replica', 'primary']) = <built-in method values of dict object at 0x7493f139ec80>()\n +      where <built-in method values of dict object at 0x7493f139ec80> = {'postgresql-k8s/0': 'replica', 'postgresql-k8s/1': 'replica', 'postgresql-k8s/2': 'primary'}.values","time":{"start":1732847969513,"stop":1732848477736,"duration":508223}},{"uid":"fca6bda61abbec61","reportUrl":"../252//#testresult/fca6bda61abbec61","status":"unknown","time":{}},{"uid":"9706dda1d7f990da","reportUrl":"../251//#testresult/9706dda1d7f990da","status":"failed","statusDetails":"AssertionError: assert 0 == 1\n +  where 0 = <built-in function countOf>(dict_values(['replica', 'replica', 'primary']), 'master')\n +    where <built-in function countOf> = operator.countOf\n +    and   dict_values(['replica', 'replica', 'primary']) = <built-in method values of dict object at 0x752e39e8c240>()\n +      where <built-in method values of dict object at 0x752e39e8c240> = {'postgresql-k8s/0': 'replica', 'postgresql-k8s/1': 'replica', 'postgresql-k8s/2': 'primary'}.values","time":{"start":1732676153843,"stop":1732676695382,"duration":541539}},{"uid":"ca76102db0365c27","reportUrl":"../250//#testresult/ca76102db0365c27","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1732588785882,"stop":1732588785882,"duration":0}},{"uid":"10d429b5083b2032","reportUrl":"../249//#testresult/10d429b5083b2032","status":"failed","statusDetails":"AssertionError: assert 0 == 1\n +  where 0 = <built-in function countOf>(dict_values(['replica', 'replica', 'primary']), 'master')\n +    where <built-in function countOf> = operator.countOf\n +    and   dict_values(['replica', 'replica', 'primary']) = <built-in method values of dict object at 0x766e94b7e300>()\n +      where <built-in method values of dict object at 0x766e94b7e300> = {'postgresql-k8s/0': 'replica', 'postgresql-k8s/1': 'replica', 'postgresql-k8s/2': 'primary'}.values","time":{"start":1732501594674,"stop":1732502131330,"duration":536656}},{"uid":"e7a38fe5b20704c2","reportUrl":"../248//#testresult/e7a38fe5b20704c2","status":"failed","statusDetails":"AssertionError: assert 0 == 1\n +  where 0 = <built-in function countOf>(dict_values(['replica', 'replica', 'primary']), 'master')\n +    where <built-in function countOf> = operator.countOf\n +    and   dict_values(['replica', 'replica', 'primary']) = <built-in method values of dict object at 0x7a1115ea67c0>()\n +      where <built-in method values of dict object at 0x7a1115ea67c0> = {'postgresql-k8s/0': 'replica', 'postgresql-k8s/1': 'replica', 'postgresql-k8s/2': 'primary'}.values","time":{"start":1732415967416,"stop":1732416515959,"duration":548543}},{"uid":"e14dfa9255800f39","reportUrl":"../247//#testresult/e14dfa9255800f39","status":"failed","statusDetails":"AssertionError: assert 0 == 1\n +  where 0 = <built-in function countOf>(dict_values(['replica', 'replica', 'primary']), 'master')\n +    where <built-in function countOf> = operator.countOf\n +    and   dict_values(['replica', 'replica', 'primary']) = <built-in method values of dict object at 0x6ffe9486c580>()\n +      where <built-in method values of dict object at 0x6ffe9486c580> = {'postgresql-k8s/0': 'replica', 'postgresql-k8s/1': 'replica', 'postgresql-k8s/2': 'primary'}.values","time":{"start":1732329575318,"stop":1732330096069,"duration":520751}},{"uid":"f71e2308c5d52623","reportUrl":"../246//#testresult/f71e2308c5d52623","status":"failed","statusDetails":"AssertionError: assert 0 == 1\n +  where 0 = <built-in function countOf>(dict_values(['replica', 'replica', 'primary']), 'master')\n +    where <built-in function countOf> = operator.countOf\n +    and   dict_values(['replica', 'replica', 'primary']) = <built-in method values of dict object at 0x740010e39cc0>()\n +      where <built-in method values of dict object at 0x740010e39cc0> = {'postgresql-k8s/0': 'replica', 'postgresql-k8s/1': 'replica', 'postgresql-k8s/2': 'primary'}.values","time":{"start":1732241730130,"stop":1732242275673,"duration":545543}},{"uid":"a06ad0f094f46871","reportUrl":"../245//#testresult/a06ad0f094f46871","status":"unknown","time":{}},{"uid":"4874ac09fb105120","reportUrl":"../244//#testresult/4874ac09fb105120","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1732070329192,"stop":1732070329192,"duration":0}}]},"tags":["unstable","asyncio","@pytest.mark.skipif(False, reason='Requires amd64 architecture')","group(1)"]},"source":"f4d7de015166ef2d.json","parameterValues":[]}